src/index.html

```diff
@@ -1,17 +1,124 @@
 <!doctype html>
 <html lang="en">
   <head>
+
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
+    
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:8080/api" method="post">
+      <fieldset class="form_input">
+        <legend>Personal information</legend>
+        <label class="PersonalInformation">
+          Surname:
+          <input 
+            type="text" 
+            name="Surname"
+            required
+          >
+        </label>
+        <label class="PersonalInformation">
+          Name:
+          <input 
+            type="text" 
+            name="Name"
+            >
+        </label>
+        <label class="PersonalInformation">
+          How old are You?
+          <input 
+            type="number" 
+            name="years" 
+            value="12"
+            min="1"
+            max="100">
+        </label>  
+        <label class="PersonalInformation">
+          Full date of birth:
+          <input type="date" name="dayOfBirh">
+        </label>  
+        <label class="PersonalInformation">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+      <fieldset class="form_input">
+        <legend>Registration</legend>
+        <label class="Registration">
+          E-mail:
+          <input 
+          type="email" 
+          name="email"
+          placeholder="email@example.com"
+          >
+        </label>
+        <label class="Registration">
+          Password:
+          <input 
+            type="password" 
+            name="password"
+            minlength="4"
+            maxlength="32" 
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form_input">
+        <legend>An interesting fact about you!</legend>
+        <label class="factAboutYou">
+          Do you love cats?
+          <input type="radio" name="love_cats">
+          Yes
+          <label>
+            <input type="radio" name="love_cats">
+            No
+          </label>
+        </label>
```

Comment:
```suggestion
        <label class="factAboutYou">
          Do you love cats?
          <input type="radio" name="love_cats">
          Yes
        </label>
        <label>
          <input type="radio" name="love_cats">
          No
       </label>
```
here you have nested labels 😢